dev/PipsPager/PipsPager.cpp

```diff
@@ -627,6 +635,11 @@ void PipsPager::OnPipsAreaBringIntoViewRequested(const IInspectable& sender, con
     }
 }
 
+void PipsPager::OnScrollViewerBringIntoViewRequested(const IInspectable& sender, const winrt::BringIntoViewRequestedEventArgs& args)
+{
+    args.Handled(true);
```

Comment:
I think both PipsPager::OnPipsAreaBringIntoViewRequested and PipsPager::OnScrollViewerBringIntoViewRequested could use some detailed comments.